scripts/lib/install

```diff
@@ -195,13 +210,35 @@ fi
 # don't run dist-upgrade in one click apps to make the
 # installation process more seamless.
 if [ -z "$NO_DIST_UPGRADE" ]; then
-    apt-get -y dist-upgrade "${APT_OPTIONS[@]}"
+    case $os_distro in
+        debian)
+            apt-get -y dist-upgrade "${APT_OPTIONS[@]}"
+            ;;
+        *)
+            echo "OS major upgraded not supported on $os_id. Skippin"
+            ;;
+    esac
 fi
 
-if ! apt-get install -y \
-    puppet git curl wget jq \
-    python python3 python-six python3-six crudini \
-    "${ADDITIONAL_PACKAGES[@]}"; then
+pkginstall_status=0
+case $os_distro in
+    debian)
+        apt-get install -y \
+            puppet git curl wget jq \
+            python python3 python-six python3-six crudini \
+            "${ADDITIONAL_PACKAGES[@]}"
+            pkginstall_status="$?"
+        ;;
+    rhel)
+        yum -y install \
+            puppet git curl wget jq \
+            python python36 python-six python36-six crudini \
```

Comment:
Hmm, for `python36`, that's something that will vary with RHEL version, right?  

Maybe something we worry about when we add RHEL 8, since it may be a while.